packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -404,6 +404,9 @@ Future<LaunchResult> _startApp(DriveCommand command, Uri webUri) async {
 
   if (webUri != null) {
     platformArgs['uri'] = webUri.toString();
+    // For web device, startApp will be triggered twice
```

Comment:
I think --start-paused only works in debug mode, so this only needs to happen in profile/release mode

Reply:
Done.